frontend_tests/node_tests/common.js

```diff
@@ -142,6 +142,92 @@ run_test("adjust_mac_shortcuts mac", ({override}) => {
     }
 });
 
+run_test("adjust_mac_mod non-mac", ({override}) => {
+    override(navigator, "platform", "Windows");
+
+    // The adjust_mac_mod has a really simple guard
+    // at the top, and we just test the early-return behavior
+    // by trying to pass it garbage.
+    common.adjust_mac_mod("not-an-array");
+});
+
+// Test default values of adjust_mac_mod
+// Expected values
+run_test("adjust_mac_mod mac expected", ({override}) => {
+    const keys_to_test_mac = new Map([
+        [["Backspace"], ["Delete"]],
+        [["Enter"], ["Return"]],
+        [["Home"], ["Fn", "←"]],
+        [["End"], ["Fn", "→"]],
+        [["PgUp"], ["Fn", "↑"]],
+        [["PgDn"], ["Fn", "↓"]],
+        [["Ctrl"], ["⌘"]],
+        [["Alt"], ["⌥"]],
+    ]);
+
+    override(navigator, "platform", "MacIntel");
+
+    const test_items = [];
+
+    for (const [old_key, mac_key] of keys_to_test_mac) {
+        const test_item = {};
+        common.adjust_mac_mod(old_key);
+
+        test_item.mac_key = mac_key;
+        test_item.adjusted_key = old_key;
+        test_items.push(test_item);
+    }
+
+    for (const test_item of test_items) {
+        assert.deepStrictEqual(test_item.mac_key, test_item.adjusted_key);
+    }
+});
+
+// Test non-default values of adjust_mac_mod
+// Random values
+run_test("adjust_mac_mod mac random", ({override}) => {
+    const keys_to_test_mac = new Map([
+        [
+            ["Ctrl", "["],
+            ["⌘", "["],
+        ],
+        [
+            ["Ctrl", "K"],
+            ["⌘", "K"],
+        ],
+        [
+            ["Shift", "G"],
+            ["Shift", "G"],
+        ],
+        [["Space"], ["Space"]],
+        [
+            ["Alt", "←"],
+            ["⌥", "←"],
+        ],
+        [
+            ["Alt", "→"],
+            ["⌥", "→"],
+        ],
+    ]);
+
+    override(navigator, "platform", "MacIntel");
+
+    const test_items = [];
+
+    for (const [old_key, mac_key] of keys_to_test_mac) {
+        const test_item = {};
+        common.adjust_mac_mod(old_key);
+
+        test_item.mac_key = mac_key;
+        test_item.adjusted_key = old_key;
+        test_items.push(test_item);
+    }
+
+    for (const test_item of test_items) {
+        assert.deepStrictEqual(test_item.mac_key, test_item.adjusted_key);
+    }
+});
+
```

Comment:
Please squash this node tests commit into the commit that adds the code being tested -- that's always the best place to put tests verifying new logic added in a series of commits.

Reply:
Fixed, thanks.